recipes/perl-sys-cpu/meta.yaml

```diff
@@ -0,0 +1,45 @@
+# This file created by conda-build 3.0.15+197.g41d59b5
+# meta.yaml template originally from:
+# /home/rdonnelly/conda/staged-recipes/recipes/perl-sys-cpu/0.61, last modified Sat Nov 11 23:55:48 2017
+# ------------------------------------------------
+#downgrade for perl 5.22 june 4th 2018
+{% set name = "perl-sys-cpu" %}
+{% set version = "0.61" %}
+{% set hash = "250a86b79c231001c4ae71d2f66428092a4fbb2070971acafd471aa49739c9e4" %}
+ 
+package:
+    name: {{ name }}
+    version: {{ version }}
+source:
+    fn: Sys-CPU-0.61.tar.gz
+    url: https://cpan.metacpan.org/authors/id/M/MZ/MZSANFORD/Sys-CPU-0.61.tar.gz
+    sha256: {{ hash }}
+build:
+    number: '0'
+    string: pl522_0
```

Comment:
You should probably not set this yourself.  Conda-build has an automatic setting that is usually good to use.  I don't think this should prevent a merge here, necessarily, it's just not good practice in general.

Reply:
I see this in many recipe . What part of this recipe should i remove please ?

Reply:
just remove the string line completely.  You are probably looking at the recipe in the package, which is a fully-rendered form that has many things baked in.  You should be referencing other raw recipes, such as the ones on conda-forge.  It is very unusual for them to set build/string.

Reply:
Ok, thanks

Reply:
It's ok now.